pom.xml

```diff
@@ -76,6 +83,30 @@
                 <artifactId>jansi</artifactId>
                 <version>${version.org.fusesource.jansi}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>${version.commons-io.commons-io}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${version.org.projectlombok.lombok}</version>
+                <scope>provided</scope>
+            </dependency>
+            <!--logging implementation for Lombok logging-->
+            <dependency>
+                <groupId>org.jboss.logging</groupId>
+                <artifactId>jboss-logging</artifactId>
+                <version>${version.org.jboss.logging.jboss-logging}</version>
+            </dependency>
+            <!--implementation of javax.json:javax.json-api-->
+            <dependency>
+                <groupId>org.glassfish</groupId>
```

Comment:
Why not https://mvnrepository.com/artifact/javax.json/javax.json-api ?

It is not necessary to promote glassfish

Reply:
Thanks! This is the reason why I placed the comment there. If you know about other vendor which implements this API, I will gladly replace this library.

Reply:
So it can't be replaced by 
```
    <groupId>javax.json</groupId>
    <artifactId>javax.json-api</artifactId>
```
?